main.tf

```diff
@@ -31,19 +31,7 @@ locals {
     }
     : {}
   )
-  lambda_endpoint = (
-    var.enable_lambda_endpoint
-    ? {
-      lambda = {
-        service             = "lambda"
-        private_dns_enabled = true
-        subnet_ids          = local.az_private_subnets
-        tags                = { Name = "lambda-vpc-endpoint" }
-      }
-    }
-    : {}
-  )
-  endpoints = merge(local.ec2_endpoint, local.lambda_endpoint)
+  endpoints = local.ec2_endpoint
```

Comment:
Since we now have just one endpoint (and I don't see us needing to add more endpoints in the future), can we remove this `endpoints` local and instead just use `local.ec2_endpoint` below?

Reply:
I was hesitating with this. It kinda made sense semantically, since endpoints could actually be empty (since its creation is conditional).

But yeah, maybe its better to leave only ec2_endpoint.

I'll upload the change as soon as I can test it.